src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -71,188 +71,141 @@ const defaultProps = {
-                                accessibilityLabel={this.props.translate('workspace.inviteMessage.personalMessagePrompt')}
-                                autoCompleteType="off"
-                                autoCorrect={false}
-                                autoGrowHeight
-                                containerStyles={[this.props.themeStyles.autoGrowHeightMultilineInput]}
-                                defaultValue={this.state.welcomeNote}
-                                value={this.state.welcomeNote}
-                                onChangeText={(text) => {
-                                    this.debouncedSaveDraf(text);
-                                    this.setState({welcomeNote: text});
-                                }}
-                            />
-                        </View>
-                    </Form>
-                </FullPageNotFoundView>
-            </ScreenWrapper>
-        );
-    }
+                    <View style={[styles.themeStyles.mv4, styles.themeStyles.justifyContentCenter, styles.themeStyles.alignItemsCenter]}>
+                        <MultipleAvatars
+                            size={CONST.AVATAR_SIZE.LARGE}
+                            icons={OptionsListUtils.getAvatarsForAccountIDs(_.values(props.invitedEmailsToAccountIDsDraft), props.allPersonalDetails, props.invitedEmailsToAccountIDsDraft)}
+                            shouldStackHorizontally
+                            shouldDisplayAvatarsInRows
+                            secondAvatarStyle={[styles.themeStyles.secondAvatarInline]}
+                        />
+                    </View>
+                    <View style={[styles.themeStyles.mb5]}>
+                        <Text>{translate('workspace.inviteMessage.inviteMessagePrompt')}</Text>
+                    </View>
+                    <View style={[styles.themeStyles.mb3]}>
+                        <InputWrapper
+                            InputComponent={TextInput}
+                            role={CONST.ROLE.PRESENTATION}
+                            inputID="welcomeMessage"
+                            label={translate('workspace.inviteMessage.personalMessagePrompt')}
+                            accessibilityLabel={translate('workspace.inviteMessage.personalMessagePrompt')}
+                            autoCompleteType="off"
+                            autoCorrect={false}
+                            autoGrowHeight
+                            inputStyle={[styles.themeStyles.verticalAlignTop]}
+                            containerStyles={[styles.themeStyles.autoGrowHeightMultilineInput]}
+                            value={welcomeMessage}
+                            onChangeText={(text) => {
+                                saveDraft(text);
+                            }}
+                            ref={(el) => {
+                                updateMultilineInputRange(el);
```

Comment:
Isn't it necessary to return early if `el` is `null` here as we do in [other components](https://github.com/Expensify/App/blob/912fa3fbb145ab903284f598e0a1c882ea819ff5/src/pages/EditRequestDescriptionPage.js#L72-L74) because `updateMultilineInputRange` cannot take `null` for [input](https://github.com/Expensify/App/blob/912fa3fbb145ab903284f598e0a1c882ea819ff5/src/libs/updateMultilineInputRange/types.ts#L3)

Reply:
Cursor and the next line appears only partially when pressing Enter now. Further pressing Enter does not display cursor. Only after deleting does the cursor appear again. Earlier it was fine.
Current behaviour: 

https://github.com/Expensify/App/assets/102477862/e8bfc1a2-ae51-418c-9457-8a3f15dc9a68

Earlier:

https://github.com/Expensify/App/assets/102477862/543627d4-f5a9-466e-a7d5-2c6f6d5953ba

Reply:
@c3024 Fixed the issue, please recheck

https://github.com/Expensify/App/assets/88325488/9bbad89a-0b1b-45ca-8434-edeefa291b1b